content/people/index.md

```diff
@@ -139,6 +139,8 @@ people:
   team:
   - core
   - highlight
+  - blog
+  description: specializes in Delphi's compute infrastructure, and is a DevOps Engineer in the Machine Learning Department at CMU.
```

Comment:
@korlaxxalrok I wrote this off the cuff, please check it and adjust as you like

Reply:
Looks good!